ret-core/src/main/kotlin/io/rabobank/ret/picocli/mixin/ContextAwareness.kt

```diff
@@ -4,13 +4,19 @@ import picocli.CommandLine
 import picocli.CommandLine.Command
 import picocli.CommandLine.Option
 
+/**
+ * Ignore Context Awareness
```

Comment:
Does not really describe the class. The value in it yes, but not this Command.

Reply:
The class is called `ContextAwareness`, the doc says "Ignore Context Awareness", that doesn't compute with me ;)

Reply:
Naming could be better indeed. Unfortunately this class doesn't really do anything with the context

Reply:
Maybe just delete this line "Ignore Context Awareness" then?

Reply:
That's probably better indeed